code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -112,6 +112,9 @@
 	buckle_lying = FALSE
 	var/static/list/can_ride_typecache = typecacheof(/mob/living/carbon/human)
 
+	var/obj/item/gps/internal // GPS location for when they die
```

Comment:
Make this an autodoc comment and mention that it does not exist until the borg is dead, and otherwise can't be referenced.

Reply:
Done.